### DISCUSSION TITLE
unable to get systemd input plugin working

### DISCUSSION THREAD

**Discussion by nareshku** [fluent/fluent-bit#5706] [Q&A]
I am running Fluent Bit v1.8.0. I am running into error `Input plugin 'systemd' cannot be loaded`. I am trying to figure out if the plugin such as `systemd` are included or do we need to do it manually?

**Comment by agup006** [comment]
Hi Naresh, are you building from source? You may need to include this parameter within the cmake  file to ensure it is part of the build
